eval_pg_utils.py

```diff
@@ -0,0 +1,145 @@
+import psycopg2
+from psycopg2.extras import Json
+import csv
+import os
+import json
+import textwrap
+
+import app_logger as log
+
+from dotenv import load_dotenv
+load_dotenv()
```

Comment:
Just as an FYI - I previously added the "override=True" since previously loaded environment variables weren't updated when code was run in subsequently.

Reply:
Good to know; I'll make sure to incorporate that in the main evals2 merge later this morning